photon-api/src/integTest/scala/com/linkedin/photon/ml/algorithm/CoordinateFactoryIntegTest.scala

```diff
@@ -139,7 +146,7 @@ object CoordinateFactoryIntegTest {
   private val INTERCEPT_INDEX = None
 
   private val OPTIMIZER_CONFIG = OptimizerConfig(OPTIMIZER_TYPE, MAX_ITER, TOLERANCE)
-  private val MOCK_NORMALIZATION = mock(classOf[NormalizationContext])
+  private val MOCK_NORMALIZATION = mock(classOf[NormalizationContext], withSettings().serializable())
```

Comment:
What was the reason for this change?

Reply:
Integ test throws an error that  `MOCK_NORMALIZATION` is not serializable. Original class `NormalizatoinContext` extends `Serializable`. Need to set `MOCK_NORMALIZATION` to be serializable as well.